tests/conftest.py

```diff
@@ -46,55 +45,68 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers", "slow: marks tests as slow (deselect with '-m \"not slow\"')"
     )
-
-
-@pytest.fixture
-def mock_database(database):
-    return database
-
-
-@pytest.fixture
-def database(request, event_loop):
-    return _database(request, event_loop, True)
+    config.addinivalue_line(
+        "filterwarnings", "ignore:Function 'semver.compare':DeprecationWarning"
+    )
 
 
 @pytest.fixture(scope='session', autouse=True)
-def global_database(request):
-    return _database(request, asyncio.get_event_loop(), False)
+async def test_data(request):
+    db = await global_database(request)
+    with open('tests/data/test-data.sql') as f:
+        async with db.acquire() as conn:
+            await conn.execute(f.read())
 
+    await db.close()
```

Comment:
Well, I wish, we work with database without dirtying it.
Like it's here.
https://github.com/norraxx/faf-aio-replayserver/blob/master/tests/fixtures/db_fixtures.py#L7

Reply:
And ofc IF we have `.commit()` in code, we should mock it.

Reply:
The database connection used during tests gets rolled back. This one is just for loading the test-data file once at the start of the session. And for some reason with the newest version of pytest-asyncio having an asyc finalizer on this session scoped fixture would throw an error due to closed event loop.

Reply:
Do we also rollback these changes?

Reply:
No, the test data is not rolled back

Reply:
`=T_T=`